addons/web/static/tests/views/list_tests.js

```diff
@@ -11787,6 +11787,49 @@ QUnit.module('Views', {
         list.destroy();
     });
 
+    QUnit.test('quickly setting row mode with owl_compatibility', async function (assert) {
+        assert.expect(1);
+
+        this.data.bar.fields.bool = { string: 'bool', type: 'boolean' };
+        this.data.foo.records[0].o2m = [1, 2, 3];
+
+        const form = await createView({
+            View: FormView,
+            model: 'foo',
+            data: this.data,
+            res_id: 1,
+            viewOptions: { mode: 'edit' },
+            arch: `<form>
+                <sheet>
+                    <notebook>
+                        <page>
+                            <field name="o2m">
+                                <tree editable="bottom">
+                                    <field name="bool"/>
+                                    <field name="display_name"/>
+                                </tree>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>`,
+        });
+
+        await testUtils.nextTick();
+
+        await testUtils.dom.click(form.$('tbody tr:eq(0) td:eq(1)'));
+        await testUtils.fields.editInput(form.$('input[name=display_name]', 'another value'));
+
+        // Double click on a field of the second row
+        testUtils.dom.click(form.$('tbody tr:eq(1) td:eq(1)'));
+        testUtils.dom.click(form.$('tbody tr:eq(1) td:eq(1)'));
+        await testUtils.nextTick();
+        await testUtils.nextTick();
+
+        assert.ok(true, 'Got here');
```

Comment:
maybe assert a more solid stuff, like the first row is saved and the second row is in edition